docs/subnets/how-to-upgrade.md

```diff
@@ -0,0 +1,469 @@
+# How To Upgrade Your Subnet-EVM
+
+## Upgrading a Subnet
+
+You can parametrize Subnet-EVM based Subnets after deployment by enabling and disabling precompiles.
+To do this, create a `upgrade.json` file and place in the appropriate directory. 
+
+This page is a tutorial on how to perform such network upgrades. It's specific for Subnet-EVM upgrades.
+
+
+[Upgrade a Subnet](./subnet-upgrade) 
+describes all the background information required regarding Subnet upgrades.
+
+:::warning
+
+It's very important that you read and understood the previously linked document. 
+Failing to do so can potentially grind your network to a halt.
+
+
+:::
+
+This tutorial assumes that you already 
+[installed](./install-avalanche-cli) Avalanche-CLI.
+It also uses `testSubnet` as a hypothetical Subnet name throughout.
```

Comment:
I would say that it assumes you have already created and deployed a subnet called `testSubnet`.